src/pinnwand/handler/api_v1.py

```diff
@@ -37,17 +55,16 @@ async def get(self) -> None:
         )
 
 
-class Paste(Base):
+class Paste(RateLimiterMixin, Base):
+    areas = ["create"]
+
     def check_xsrf_cookie(self) -> None:
         return
 
     async def get(self) -> None:
```

Comment:
There's a slight change in behaviour here, as making a request to this would count towards the create ratelimit, which wouldn't really be an issue but doesn't feel quire right.